docs/fleet/provision.md

```diff
@@ -1,278 +1,186 @@
 
-- To restart the Viam Agent:
+{{< tabs >}}
+{{% tab name="Without pre-configured network" min-height="703px" %}}
 
-  {{< alert title="Alert" color="note" >}}
-  When you restart the Viam Agent, the agent will restart `viam-server` as well.
-  {{< /alert >}}
+{{<video webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with viam-agent." poster="/platform/provisioning-demo.jpg" class="alignright" max-width="400px">}}
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl restart viam-agent
-  ```
+1. When a machine that has `viam-agent` installed and `agent-provisioning` configured comes online, `agent-provisioning` creates a WiFi hotspot.
 
-- To completely uninstall the Viam Agent and `viam-server`, run the following command:
+   - The [`agent-provisioning` configuration](#configuration) is at <file>/etc/viam-provisioning.json</file.
+   - The [`viam-agent` configuration](/configure/agent/#configuration), if provided, is at <file>/etc/viam.json</file> alongside the rest of the machine configuration, including which fragments to use (if any).
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/uninstall.sh)"
-  ```
+1. The end user uses their mobile device or computer and connects to the WiFi hotspot.
 
-  This command uninstalls Viam Agent, `viam-server`, the machine configuration file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
+   - By default, the hotspot network is named `viam-setup-HOSTNAME`, where `HOSTNAME` is replaced with the hostname of your machine.
+     The WiFi password for this hotspot network is `viamsetup` by default.
+     You can customize these values in the machine's or fragment's [`viam-agent` configuration](/configure/agent/#configuration).
 
-  For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
+1. If the end user has a provisioning web or mobile app, they should now open that app to complete setup.
+   Alternatively, you will be redirected to a sign-in page.
+   If you are using a laptop or are not redirected, try opening [http://viam.setup/](http://viam.setup/) in a browser.
 
-### View Viam Agent logs
+1. In the browser or web or mobile app, the user will be prompted to provide the network information for the machine, that is the primary network for the machine.
 
-The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
-You can find these messages on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
+   - If there is no machine configuration at <file>/etc/viam.json</file>, the provisioning app or captive portal will also require the user to paste an app configuration file.
```

Comment:
I think that's something we need to chat about further because we can't have a `viam-server` config and a `viam-server` configuration file. That's too confusing...